docs/topics/settings.txt

```diff
@@ -16,11 +16,15 @@ Here are a couple of example settings::
     ALLOWED_HOSTS = ['www.example.com']
     DEBUG = False
     DEFAULT_FROM_EMAIL = 'webmaster@example.com'
+    INSTALLED_APPS=[
+    "myapp"
+    ]
 
 .. note::
 
     If you set :setting:`DEBUG` to ``False``, you also need to properly set
     the :setting:`ALLOWED_HOSTS` setting.
+    INSTALLED_APPS is used to register our apps in a project and register modules we are using in our project
```

Comment:
This page is a topic about `settings` not a reference. I don't think we need any more settings in this short example.